hawc/apps/study/filterset.py

```diff
@@ -58,31 +54,16 @@ def filter_queryset(self, queryset):
             query &= Q(published=True)
         return queryset.filter(query)
 
-    def filter_citation(self, queryset, name, value):
-        query = Q(short_citation__icontains=value) | Q(full_citation__icontains=value)
+    def filter_search(self, queryset, name, value):
+        query = (
+            Q(short_citation__icontains=value)
+            | Q(full_citation__icontains=value)
+            | Q(identifiers__unique_id__icontains=value)
+        )
         return queryset.filter(query)
 
     def filter_data_type(self, queryset, name, value):
         return queryset.filter(**{value: True})
 
     def filter_assigned_user(self, queryset, name, value):
         return queryset.filter(riskofbiases__author=value, riskofbiases__active=True).distinct()
-
-    def create_form(self):
-        form = super().create_form()
-        form.fields["assigned_user"].queryset = self.assessment.pms_and_team_users()
-        if not self.include_rob_authors:
-            form.fields.pop("assigned_user")
-        if not self.perms["edit"]:
-            form.fields.pop("published")
```

Comment:
We need this logic too. You could move it to get_filterset_form_kwargs as a dynamic_fields argument, but it can probably stay here if it works as is